livingdoc-repositories/src/main/kotlin/org/livingdoc/repositories/config/ConfigUtils.kt

```diff
@@ -0,0 +1,29 @@
+package org.livingdoc.repositories.config
+
+import org.yaml.snakeyaml.Yaml
+import java.io.FileNotFoundException
+import java.net.URL
+import kotlin.reflect.KClass
+
+object ConfigUtils {
+
+    private val yaml = Yaml()
+
+    @Throws(FileNotFoundException::class)
+    fun loadFileFromClasspath(filePath: String): Map<String, Any> {
+        val file = getFile(filePath) ?: throw FileNotFoundException("File not found: $filePath")
+        return file.openStream().use {
+            yaml.load(it) as Map<String, Any>
+        }
+    }
+
+    fun <T : Any> toObject(configData: Map<String, Any>, type: KClass<T>): T {
+        val yamlString = yaml.dump(configData)
```

Comment:
Necessary for reading parts of the original YAML structure (e.g. the `config` node) as objects. There is no read `Map<String, Any>` as object method.